Language/Haskell/GhcMod/Monad/Types.hs

```diff
@@ -78,44 +84,44 @@ module Language.Haskell.GhcMod.Monad.Types (
 #define MONADIO_INSTANCES 1
 #endif
 
-import Language.Haskell.GhcMod.Types
```

Comment:
If you're going to so major reformatting like this please do it in a separate commit. I don't like this style of imports though (also it's inconsistent with the rest of the code base) so please remove this part of the patch.